2023-01-03_infSILAC_Processing_and_QC_of_SILAC_data.Rmd

```diff
@@ -436,38 +393,21 @@ print(pData(pep_res))
 #> data frame with 0 columns and 4 rows
 ```
 
+
 ```{r}
 # Add replicate number to phenotype data
-pData(pep_res)$Replicate <- 1:ncol(exprs_data)
+pData(pep_res)$Replicate <- as.numeric(colnames(exprs_data))
```

Comment:
Replicate order in `exprs_data` is not numerical, it's 1, 10, 11, 12, ... (!) Better to define from the data to avoid issues like this.